vendor-extra/ViewsBundle/src/Views/InlineViewConverterRegistry.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ViewsBundle\Views;
+
+use FluentDOM\DOM\Element;
+use FluentDOM\DOM\Node\NonDocumentTypeChildNode;
+use function array_map;
+use function array_push;
+use function iterator_to_array;
+
+final class InlineViewConverterRegistry implements InlineViewConverter
+{
+    private $visitors = [];
+
+    public function add(InlineViewConverterVisitor ...$visitors) : void
+    {
+        array_push($this->visitors, ...$visitors);
+    }
+
+    public function convert(NonDocumentTypeChildNode $object, array $context = []) : View
+    {
+        $view = new View(null, []);
+
+        foreach ($this->visitors as $visitor) {
+            $view = $visitor->visit($object, $view, $context);
+        }
+
+        if (!$view->getTemplate()) {
+            return new View('@LiberoPatterns/text.html.twig', ['nodes' => (string) $object]);
```

Comment:
This couples the `ViewsBundle` to the `LiberoPatternsBundle`, should probably throw an exception instead and have a late running listener somewhere else. Can leave for now though.